stable/pipelines/templates/argo/workflow-controller-configmap.yaml

```diff
@@ -6,6 +6,23 @@ metadata:
   labels:
     component: workflow-controller
 {{ include "pipelines.commonLabels" . | indent 4 }}
+{{- if and (not .Values.iguazio) (.Values.minio.enabled) }}
```

Comment:
instead of that, use "backend" / "storageMode" and use "kind: v3io" / "kind: minio" because i dont see how they both go together, and that entire PR is to provide another flow for pipelines to work against minio.

Reply:
add to values:
storageMode:
  kind:
    v3io:
      enabled: true
    minio:
      enabled: false